ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1214,22 +1261,31 @@ setOptions({
                 start: function (
                     this: NavigationBindings,
                     e: PointerEvent
-                ): Annotation {
+                ): Annotation|void {
                     var coords = this.chart.pointer.getCoordinates(e),
+                        coordsX = this.getAssignedAxis(coords, 'x'),
+                        coordsY = this.getAssignedAxis(coords, 'y'),
                         navigation = this.chart.options.navigation;
 
+                    // Exit if clicked out of axes area
```

Comment:
I would add the same comment everywhere you have that `if`. Not everyone is looking through the whole file, very often we concentrate on one method and I think we should have similar comments.